crates/mysten-util-mem/Cargo.toml

```diff
@@ -14,7 +14,7 @@ cfg-if = "1.0.0"
 hashbrown = { version = "0.12", optional = true }
 mysten-util-mem-derive = { path = "../mysten-util-mem-derive", version = "0.1" }
 impl-trait-for-tuples = "0.2.0"
-fastcrypto = { git = "https://github.com/MystenLabs/fastcrypto", rev = "4b7a06fe393003013489990dc2f4d3d2ebb32539" }
+fastcrypto = { git = "https://github.com/MystenLabs/fastcrypto", rev = "ab44e914084bd2fa65e7caf64ab32d61910a5d30" }
```

Comment:
This does not match https://github.com/MystenLabs/sui/blob/127e0929b0317ee40a4f0cd95825db25616ca1d5/Cargo.toml#L96 am I missing something?

Reply:
yes, I plan to update sui as well - https://github.com/MystenLabs/sui/pull/5533

Reply:
That's true. This PR will be closed, and we will merge https://github.com/MystenLabs/sui/pull/5533 instead.